proposals/839-outreach-committee/kubeflow-outreach-committee.md

```diff
@@ -0,0 +1,31 @@
+# KEP- 839: Proposal for the Creation of the Kubeflow Outreach Committee (KOC)
+
+## Overview
+
+We propose the creation of the **Kubeflow Outreach Committee (KOC)** to lead efforts in growing and engaging the broader AI/ML community, not just the Kubeflow community. This committee will be responsible for fostering visibility, engagement, and inclusivity, as well as helping to expand Kubeflow’s presence and user base.
+
+## Objectives
+
+- Lead efforts to grow and engage the AI/ML community through events, educational materials, and advocacy for Kubeflow.
+- Onboard new contributors and users to Kubeflow, particularly from diverse and underrepresented communities.
+- Collaborate with other committees and working groups to align outreach activities with the broader goals of the Kubeflow project.
+- Foster a welcoming, inclusive, and diverse community within Kubeflow.
+
+## Committee Structure
+
+- **Appointment of Members**: Members will be appointed by the Kubeflow Steering Committee (KSC) at their discretion. Members can also be removed for violating the CNCF Code of Conduct.
+- **KOC Co-Leads**: Co-leads will be established Kubeflow community members, with new members being appointed from both within and outside of the Kubeflow community. The co-leads will focus on driving growth and engagement.
+- **Expansion**: The KOC will have the authority to appoint new members with the approval of one KSC member.
```

Comment:
Do we really need to have Co-Leads and Expansion ? Maybe having members appointed by KSC + Expansion is sufficient initially ?

Reply:
I think we wanted expansion for early career participants

Reply:
I think it needs to be clarified that "members" are non voting for the purposes of KOC decisions.

Only leads have a vote, and there can only be 3 leads (as listed in the charter above).

Reply:
Can you clarify what will be each's role responsibility?
I'd start with the members and let this group define the expansion as needed.